packages/http-server/src/server.ts

```diff
@@ -1,11 +1,30 @@
 import { configMergerFactory } from '@stoplight/prism-core';
 import { createInstance, IHttpMethod, TPrismHttpInstance } from '@stoplight/prism-http';
+import * as buildSerializer from 'fast-json-stringify';
 import * as fastify from 'fastify';
 import { IncomingMessage, Server, ServerResponse } from 'http';
-
 import { getHttpConfigFromRequest } from './getHttpConfigFromRequest';
 import { IPrismHttpServer, IPrismHttpServerOpts } from './types';
 
+const problemJsonSerializer = buildSerializer({
```

Comment:
Maybe add a comment that this should be in sync with the JsonProblemError type?

Reply:
I was considering the idea of ditching this completely. This is only required to speedup the error responses but I think the effort of keeping this in sync and the code required to set this up is not worth the effort. What do you think?

Reply:
Yeah, let's get rid of that 531ff60